xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 11.4.0
+JSONRPC_VERSION 11.5.0
```

Comment:
You cross-posted with the merge of 11.5.0, so 11.6.0 here

Reply:
Done.